app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
Same here, please implement your association using this approach `belongs_to :author, class_name: 'User'`